modules/migration-using-must-gather.adoc

```diff
@@ -19,13 +27,33 @@ The `openshift-migration-must-gather-rhel8` image for {mtc-short} collects migra
 ----
 $ oc adm must-gather --image=registry.redhat.io/rhmtc/openshift-migration-must-gather-rhel8:v{mtc-version}
 ----
++
+The data is saved in a local `must-gather` directory.
+
+.Viewing metrics data with Prometheus
 
-. Remove authentication keys and other sensitive information.
-. Create an archive file containing the contents of the `must-gather` data directory:
+. Run the `oc adm must-gather` command with the `gather_metrics_dump` script:
 +
 [source,terminal]
 ----
-$ tar cvaf must-gather.tar.gz must-gather.local.<uid>/
+$ oc adm must-gather --image={must-gather} -- /usr/bin/gather_metrics_dump
 ----
++
+[NOTE]
+====
+This process might take a long time. `must-gather` processes the most recent `prom_data.tar.gz` file in the `/must-gather/metrics` directory.
+====
 
-. Upload the compressed file as an attachment to your customer support case.
+. Create and run a local Prometheus instance to display the dumped data:
++
+[source,terminal]
+----
+$ make prometheus-run
+----
+
```

Comment:
I just noticed that a step is missing. You will need to clean it up because it is copied from BZ.

8. open Prometheus web console http://localhost:9090 by browser

Reply:
Is [this](https://docs.openshift.com/container-platform/4.7/monitoring/accessing-third-party-uis.html#accessing-third-party-uis-using-the-cli_accessing-third-party-uis) relevant here?

Reply:
Yes. Otherwise, how are they going to see the info? It's a critical step. There is a ton of data and the user is not going to grep all the directories to find what they're looking for.

Reply:
But they have a local instance of Prometheus. Why do they need its web UI?

Reply:
Because that's how they see the data. Prometheus is running but you access the console with a browser.